recipes/rn-key-cache-dependencies.md

```diff
@@ -0,0 +1,37 @@
+# (React Native) Cache NPM dependencies (Beta)
+
+## Description
+
+Cache the contents of the `node_modules` folder with the new key-based caching steps, **Save Cache** and **Restore Cache**.
+
+## Instructions
+
+1. Add the [Restore cache](https://github.com/bitrise-steplib/bitrise-step-restore-cache) Step to the workflow.
+1. For the **Cache keys** input, add the following keys:
+    ```
+    npm-cache-{{ checksum "package-lock.json" }}
+    npm-cache-
+    ```
+    The first key will result in a unique string based on the exact dependencies defined in `package-lock.json` (make sure to commit the file!). If there is no cache to restore with that key, the Step will use the second key and will restore a cache with a key that starts with `npm-cache-`. This will result in a cache that was saved for slightly different dependencies, but it's still better than not restoring any cache.
```

Comment:
```suggestion
    The first key will result in a unique string based on the exact dependencies defined in `package-lock.json` (make sure to commit the file!). If there is no cache to restore with that key, the Step will move on to the second key and will restore a cache with a key that starts with `npm-cache-`. This will result in a cache that was saved for slightly different dependencies, but it's still better than not restoring any cache.
```